### PR TITLE
Feat/dist points mesh

### PR DESCRIPTION
**Pull request by Tanguylo**
* **Please check if the PR fulfills these requirements**
- [x] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix:
- [x] New features: Allow to compute distances from mesh to points
- [ ] Documentation
- [ ] Optimization

### CODE REVIEW

**Comment by Tanguylo** [comment]
The developments will pass checks when BSpline2D_fixes will pass them.

File: volmdlr/templates.py [hunk 1]
```diff
@@ -108,15 +108,15 @@
...
-         })
+        //camera.onViewMatrixChangedObservable.add(() => {
+        //    if ((Date.now() - scene.lastEdgewidthUpdate) > 1000){
+        //        scene.lastEdgewidthUpdate = Date.now();
+        //        for (mesh of scene.meshes){
+        //            var dist = BABYLON.Vector3.Distance(camera.position, mesh.position);
+        //            mesh.edgesWidth = dist*0.1;
+        //        }
+        //    }
+        // })
```
Comment (masfaraud) [inline volmdlr/templates.py:119]: Why disabling dynamic edge width?

File: volmdlr/faces.py [hunk 2]
```diff
@@ -8899,6 +8902,30 @@ def from_mesh_data(cls, positions, faces):
             triangles.append(Triangle3D(points[i1], points[i2], points[i3]))
         return cls(triangles)
 
+    def points_cloud_distances(self, points_cloud: List[volmdlr.Point3D]) \
```
Comment (masfaraud) [inline volmdlr/faces.py:8905]: There are some point cloud classes in volmdlr/cloud.py
Why not using them?

File: volmdlr/templates.py (hunk 1 above)
Comment (Tanguylo) [reply volmdlr/templates.py:119]: This is from the branch BSpline2D_fixes on which OpenTriangleShell3D is firstly created. Maybe I should change the destination branch from dev to BSpline2D_fixes ?

File: volmdlr/faces.py (hunk 2 above)
Comment (Tanguylo) [reply volmdlr/faces.py]: Done in last commit : Moved method to PointCloud3D and add a method to generate a trimesh from ClosedTriangleShell3D

**Review by WirajanDASILVA — APPROVED** [review]

**Review by WirajanDASILVA — APPROVED** [review]

**Pull request merged**